Makefile

```diff
@@ -38,6 +34,59 @@ test/unit:
 test/compile:
 	@GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go test -c -o=$(TEST_COMPILE_OUTPUT) ./test/e2e/...
 
+##############################
+# CI                         #
+##############################
+
+.PHONY: code/compile
+code/compile: code/gen
+	@GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o=$(CODE_COMPILE_OUTPUT) ./cmd/manager/main.go
+
+.PHONY: setup/travis
+setup/travis:
+	@echo Installing dep
+	curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+	@echo Installing Operator SDK
+	curl -Lo ${GOPATH}/bin/operator-sdk https://github.com/operator-framework/operator-sdk/releases/download/v0.7.0/operator-sdk-v0.7.0-x86_64-linux-gnu
+	chmod +x ${GOPATH}/bin/operator-sdk
+	@echo setup complete
+
+##############################
+# Application                #
+##############################
+
+.PHONY: install
+install:
```

Comment:
This seems very similar to the `cluster/prepare` task, and `uninstall` seems like `cluster/clean`. Is there a benefit to having both? Would it make sense for one to inherit from the other?

Reply:
Hi @grdryn,

Following a better explanation:
- The install as the name suggested is installing the product. 
- The uninstall as the same suggested uninstall the product and undo all actions performed in the install. 

Regards the pre-existent ones, I did not would like to remove them now in this PR and we can improve them. I agree with you that the cluster/clean is, in fact, uninstall all and not make too much sense. However, 
the `cluster/prepare` install all examples CR for development purpose only. I think we could improve this name to make it more clear.

Reply:
> Can we move here and do the next PR to check these pieces?

+1